CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -714,44 +714,26 @@ public int RecordDisplayCommandBuffer(VkCommandBufferInheritanceInfo inheritance
         CVKAssertNotNull(cvkSwapChain);
         cvkVisualProcessor.VerifyInRenderThread();
         int ret;
-        
-        try (MemoryStack stack = stackPush()) {
-            
-            CVKCommandBuffer commandBuffer = commandBuffers.get(imageIndex);
-            CVKAssertNotNull(commandBuffer);
-            CVKAssertNotNull(pipelines.get(imageIndex));
-            
-            commandBuffer.BeginRecordSecondary(VK_COMMAND_BUFFER_USAGE_RENDER_PASS_CONTINUE_BIT,
-                                                           inheritanceInfo);
-            
-            // Set the dynamic viewport and scissor
-            commandBuffer.viewPortCmd(cvkSwapChain.GetWidth(), cvkSwapChain.GetHeight(), stack);
-            commandBuffer.scissorCmd(cvkVisualProcessor.GetCanvas().GetCurrentSurfaceExtent(), stack);
-            
-            // Bind the graphics pipeline
-            commandBuffer.BindGraphicsPipelineCmd(pipelines.get(imageIndex));
-            
+                    
+        CVKCommandBuffer commandBuffer = commandBuffers.get(imageIndex);
```

Comment:
These should be renamed to displayCommandBuffers for all renderables I think.

Reply:
Renamed